### ISSUE TITLE
Meta-Learning via Learned Loss. (arXiv:1906.05374v4 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#2947]
Meta-Learning via Learned Loss. (arXiv:1906.05374v4 [cs.LG] UPDATED)<br>
https://ift.tt/2mKApAx<br>
<p>Typically, loss functions, regularization mechanisms and other important aspects of training parametric models are chosen heuristically from a limited set of options. In this paper, we take the first step towards automating this process, with the view of producing models which train faster and more robustly. Concretely, we present a meta-learning method for learning parametric loss functions that can generalize across different tasks and model architectures. We develop a pipeline for meta-training such loss functions, targeted at maximizing the performance of the model trained under them. The loss landscape produced by our learned losses significantly improves upon the original task-specific losses in both supervised and reinforcement learning tasks. Furthermore, we show that our meta-learning framework is flexible enough to incorporate additional information at meta-train time. This information shapes the learned loss function such that the environment does not need to provide this information during meta-test time. We make our code available at https://ift.tt/3irEAd3.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
